contracts/SlashIndicator.sol

```diff
@@ -158,6 +188,73 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function handleFinalityReport(FinalityEvidence calldata _evidence) external onlyInit oncePerBlock noReentrant {
+    require(
+      _evidence.numA + finalityDistance >= _evidence.numB || _evidence.numB + finalityDistance >= _evidence.numA,
+      'too long distance between blocks'
+    );
+    require(
+      _evidence.numA <= block.number &&
```

Comment:
change to `_evidence.numA < block.number` and `_evidence.numB < block.number`?

Reply:
Fixed